core/src/init/button/size.module.css

```diff
@@ -0,0 +1,33 @@
+.large {
+	height: 40px;
+	padding: 9px 16px;
+	font-size: 16px;
+}
+
+.largeIcon {
+	height: 40px;
+	width: 40px;
+	padding: 12px;
```

Comment:
Should not have padding defined here as the width and height should be enough already. The icon is always aligned at the center.